### ISSUE TITLE
a11y issues

### ISSUE THREAD

**Issue by ctcpip** [tc39/tc39.github.io#327]
we need to fix a11y issues -- on the home page, there are 38 contrast issues and one svg missing alt text.  there are issues with other pages as well.  I will add a checklist here in the near future

**Comment by ctcpip** [comment]
the spec page breaks the scanner :)  https://github.com/dequelabs/axe-core/issues/4000

**Comment by ctcpip** [comment]
![Screenshot 2023-07-10 at 10 57 27 AM](https://github.com/tc39/tc39.github.io/assets/15326526/e9bf66b5-a48c-4d9c-8e19-5ac634f1f508)

huge thanks to @smorimoto for fixing!

**Comment by smorimoto** [comment]
From my own experience, accessibility is always important!
